c-bindings-gen/README.md

```diff
@@ -0,0 +1,12 @@
+LDK C Bindings Generator
+========================
+
+This program parses a Rust AST from a single-lib.rs Rust crate (passed in on stdin) and generates a
```

Comment:
suggested phrasing: This program compiles all of Rust-Lightning's `lightning` crate into a Rust crate with one file, then parses the Rust AST from that file (passed in on stdin) and generates...

Reply:
Hmm, in theory its crate-agnostic (though with some specific requirements), and the one-file generation is in `genbindings.sh`, which calls the relevant `cargo` subcommand.

Reply:
Ah, ok. I just think the `single-lib.rs Rust crate` phrase is a bit oblique. 

Maybe: "This program parses a Rust AST from a Rust crate with a single file (passed in on stdin) ..." and then maybe a sentence at the end: "`genbindings.sh` is also responsible for compiling the `lightning` crate into a single file for consumption by the `lightning-c-bindings` crate."